### DISCUSSION TITLE
Kubernetes sample don't create ingress

### DISCUSSION THREAD

**Discussion by andreatosato** [dotnet/yarp#1961] [Q&A] [answered]
Hi, I forked repository and try "combinated" solution.
I obtain AN service that had "loadbalancer". I haven't any ingress.
Can anyone explain me how can I run yarp as ingress?
Thanks a lot

**Comment by MihaZupan** [comment, marked as answer]
Hey, did you encounter any errors while following the steps in https://github.com/microsoft/reverse-proxy/tree/main/samples/KubernetesIngress.Sample/Combined?

Did you also deploy the sample backend app and apply its ingress resource https://github.com/microsoft/reverse-proxy/tree/main/samples/KubernetesIngress.Sample/backend?
